### PR TITLE
doc improvement: update edge Impulse user guide

### PR DESCRIPTION
**Pull request by divipillai**
update the Edge Impulse user guide to match
with the integration template

NCSDK-19384

### CODE REVIEW

File: doc/nrf/external_comp/edge_impulse.rst [hunk 1]
```diff
@@ -10,28 +10,48 @@ Edge Impulse integration
 `Edge Impulse`_ is a development platform that can be used to enable `embedded machine learning`_ on |NCS| devices.
 You can use this platform to collect data from sensors, train machine learning model, and then deploy it to your Nordic Semiconductor's device.
 
-Overview
-********
+nRF Machine learning application can also be used to collect data from devices (For example, it supports Thingy:53 with an out-of-the-box accelerometer).
```
[the remark this answers is not in the document]
Comment (MarekPieta) [reply doc/nrf/external_comp/edge_impulse.rst:13]: You actually describe an applications that is part of the NCS

[the remark this answers is not in the document]
Comment (divipillai) [reply doc/nrf/external_comp/edge_impulse.rst:13]: Agree, I will move to Applications and samples section.

[the remark this answers is not in the document]
Comment (MarekPieta) [reply doc/nrf/external_comp/edge_impulse.rst:13]: Thanks

File: doc/nrf/external_comp/edge_impulse.rst [hunk 2]
```diff
@@ -10,28 +10,48 @@ Edge Impulse integration
...
+
+Integration prerequisites
+*************************
+
+Before you start the |NCS| integration with Edge Impulse, make sure that the following prerequisites are completed:
+
+   * :ref:`Setup of nRF Connect SDK <getting_started>` environment.
+   * Setup of the required :term:`Development Kit (DK)`.
+   * An `Edge Impulse studio account <Edge Impulse studio signup_>`_ and an Edge Impulse project.
```
Comment (annwoj) [inline doc/nrf/external_comp/edge_impulse.rst:22]:
```suggestion
   * Creation of an `Edge Impulse studio account <Edge Impulse studio signup_>`_ and an Edge Impulse project.
```

File: doc/nrf/external_comp/edge_impulse.rst [hunk 3]
```diff
@@ -10,28 +10,48 @@ Edge Impulse integration
...
-**************************************
-
 The machine learning model is distributed as a single :file:`zip` archive that includes C++ library sources.
 This file is used by the |NCS| build system to build the Edge Impulse library.
-See the following section for steps required to generate this archive and add it to the build system.
+
+Integration steps
+*****************
+
+Complete the following steps to generate this archive and add it to the build system:
```
Comment (annwoj) [inline doc/nrf/external_comp/edge_impulse.rst:46]:
```suggestion
Complete the following steps to generate the archive and add it to the build system:
```

File: doc/nrf/external_comp/edge_impulse.rst [hunk 4]
```diff
@@ -163,3 +187,22 @@ Complete the following steps to download the model from a public Edge Impulse pr
 
    .. note::
       This URI includes the ``modelType=int8`` parameter because from public Edge Impulse projects you can only download quantized models created with Edge Impulse's EON Compiler.
+
+Applications and samples
+************************
+
+The following application uses the Edge Impulse integration in |NCS|:
+
+* :ref:`nrf_machine_learning_app` - demonstrates the complete out of the box reference design.
```
Comment (annwoj) [inline doc/nrf/external_comp/edge_impulse.rst:196]:
```suggestion
* :ref:`nrf_machine_learning_app` - demonstrates a complete out of the box reference design.
```

File: doc/nrf/external_comp/edge_impulse.rst [hunk 5]
```diff
@@ -163,3 +187,22 @@ Complete the following steps to download the model from a public Edge Impulse pr
...
+
+The following samples demonstrate the Edge Impulse integration in |NCS|:
```
Comment (annwoj) [inline doc/nrf/external_comp/edge_impulse.rst:198]:
```suggestion
The following samples demonstrate the Edge Impulse integration in the |NCS|:
```

File: doc/nrf/external_comp/edge_impulse.rst [hunk 6]
```diff
@@ -163,3 +187,22 @@ Complete the following steps to download the model from a public Edge Impulse pr
 
    .. note::
       This URI includes the ``modelType=int8`` parameter because from public Edge Impulse projects you can only download quantized models created with Edge Impulse's EON Compiler.
+
+Applications and samples
+************************
+
+The following application uses the Edge Impulse integration in |NCS|:
```
Comment (annwoj) [inline doc/nrf/external_comp/edge_impulse.rst:194]:
```suggestion
The following application uses the Edge Impulse integration in the |NCS|:
```

**Review by emob-nordic — CHANGES_REQUESTED** [review]

File: doc/nrf/external_comp/edge_impulse.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (divipillai) [reply doc/nrf/external_comp/edge_impulse.rst:13]: @emob-nordic : As per the comment here: I plan to move the sentence to https://github.com/nrfconnect/sdk-nrf/pull/10936#discussion_r1191197145, applications and samples. Hope this works for you.

[the remark this answers is not in the document]
Comment (emob-nordic) [reply doc/nrf/external_comp/edge_impulse.rst:13]: Oh. I didn't notice that. That works for me.

[the remark this answers is not in the document]
Comment (emob-nordic) [reply doc/nrf/external_comp/edge_impulse.rst:13]: Let me know once, moved so i could rereview.

**Review by MarekPieta — APPROVED** [review]

**Review by annwoj — APPROVED** [review]

**Review by emob-nordic — APPROVED** [review]

**Pull request merged**